src/style.css

```diff
@@ -0,0 +1,89 @@
+/* Fonts */
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+/* Page */
+:root {
+  --border-color: #e1e7ed;
+  --input-text-color: #3d4e61;
+  --icon-search-small: 11px;
+  --icon-search-large: 19px;
+}
+
+.page {
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+}
+
+/* Search Form */
+.search-form {
+  margin: 20px 0;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.input[type="search"] {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
`font-family: "Avenir", sans-serif` is duplicated
you need to set font-family only one time